commons/zenoh-config/src/lib.rs

```diff
@@ -147,6 +147,9 @@ validated_struct::validator! {
             GossipConf {
                 /// Whether gossip scouting is enabled or not.
                 enabled: Option<bool>,
+                /// When true, gossip scouting informations are propagated multiple hops to all nodes in the local network.
+                /// When flase, gossip scouting informations are only propagated to the next hop.
```

Comment:
Typo: `When flase` -> `When false`

Reply:
Don't forget to align `lib.rs` and `DEFAULT_CONFIG.json5`'s documentations

Reply:
Done